src/pages/RequestsIndex/base-search-api.ts

```diff
@@ -47,6 +47,13 @@ const defaultColumnsData: ColumnData[] = [
   ...unsearchableColumns,
 ]
 
+/** Fetch number of data rows per page from storage so we can persist the value */
+const getPageSize = () => {
+  const initialState = window.localStorage.getItem('tableState:Requests')
+  const pageSize = initialState ? JSON.parse(initialState).pageSize : 10
```

Comment:
It would be a pretty severe refactor in order to get this to work with the hook. I don't feel that it's worth the effort. So the best answer to my original question,

- "*Could the useLocalStorage hook be used instead of these calls to window.localStorage?*"

is "No" as far as I'm concerned. That's just the way it goes sometimes... ;-)